src/MongoDB.Driver.Core/Core/Operations/ChangeStreamOperation.cs

```diff
@@ -265,7 +259,12 @@ public BsonDocument ResumeAfter
             set { _resumeAfter = value; }
         }
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Gets or sets the start after value.
+        /// </summary>
+        /// <value>
+        /// The start after value.
+        /// </value>
```

Comment:
You can use:

    /// <inheritdoc />

Reply:
Currently, yes. Done

Reply:
Sorry, It was accidentally rollbacked somehow. Done